packages/gafl-webapp-service/src/processors/api-transaction.js

```diff
@@ -13,8 +13,9 @@ const prepareApiTransactionPayload = async request => {
     permissions: transactionCache.permissions.map(p => {
       const permission = {
         permitId: p.permit.id,
-        licensee: Object.assign((({ countryCode, ...l }) => l)(p.licensee),
-          { country: countriesList.find(c => c.code === p.licensee.countryCode).name }),
+        licensee: Object.assign((({ countryCode, ...l }) => l)(p.licensee), {
```

Comment:
I don't think we should be making arbitrary format changes to each others code.

Reply:
As above - prettier-standard picked this up